teps/README.md

```diff
@@ -75,7 +75,7 @@ The TEP `OWNERS` are the **main** owners of the following projects:
 To create a new TEP, use the [teps script](./tools/README.md):
 
 ```shell
-$ ./teps.py new --title "The title of the TEP" --author nick1 --author nick2
+$ ./teps/tools/teps.py new --title "The title of the TEP" --author nick1 --author nick2
```

Comment:
This file is generated, and it should not be changed directly.
Could you change this in the [template](https://github.com/tektoncd/community/blob/main/teps/README.md.mustache) instead?